configs/addons/counterstrikesharp/gamedata/gamedata.json

```diff
@@ -265,5 +265,11 @@
       "windows": 584,
       "linux": 584
     }
+  },
+  "CServerSideClient": {
+     "offsets": {
+       "windows": 8,
+       "linux": -64
+     }
```

Comment:
#563 might need a revision

Reply:
Do I need change the game data? I think the offset of both os is correct and I have test it.